openapi.yaml

```diff
@@ -3909,18 +3911,17 @@ paths:
           curl -H "Content-Type: application/json" \
               -H "Authorization: Bearer $TOKEN" \
               -X POST -d '{
-                "description": "This is an example Machine Image.",
-                "label": "example-machine-image-label",
+                "description": "Optional details about the Image",
+                "label": "Example Image",
                 "region": "us-east"
               }' \
               https://api.linode.com/v4/images/upload
       - lang: CLI
         source: >
-          linode-cli image-upload \
+          linode-cli images upload \
+            --description "Optional details about the Image" \
             --label "Example Image" \
-            --description "Some details about the image" \
-            --region us-east \
-            ~/Downloads/image-file.img.gz
+            --region us-east
```

Comment:
Personally, I'd still leave the plugin invocation in the example here - it's more of the correct way to do it.

Reply:
Thanks for the suggestion -- went with including both commands